cosmoz-sse.js

```diff
@@ -0,0 +1,127 @@
+// @license Copyright (C) 2019 Neovici AB - Apache 2 License
+const SSEHandlers = {
+	text: data => data,
+	json: data => JSON.parse(data)
+};
+const { enqueueDebouncer, Debouncer } = Polymer;
+
+class CosmozSSE extends Polymer.Element {
+	static get is() {
+		return 'cosmoz-sse';
+	}
+
+	static get properties() {
+		return {
+			url: {
+				type: String
+			},
+			events: {
+				type: Array,
+				value: []
+			},
+			handleAs: {
+				type: String,
+				value: 'json'
+			}
+		};
+	}
+
+	static get observers() {
+		return ['_onChange(url, events)'];
```

Comment:
Unless there's something I miss, I would prefer that url has an observer which:
1. Removes event listeners
2. Disconnects
3. Connects
4. Adds event listeners

While events has its own observer that does 1 and 4 without affecting the connection.

Any reason why we debounce?
I'd rather see that the observer should be the connect method, and the url-check would be there.

Reply:
You are right that the debounce was unnecessary. Separating the observer in two helped me discover a more graceful way of handling disconnecting/re-connecting the element to the document.